tests/src/test/scala/cats/tests/RegressionSuite.scala

```diff
@@ -102,11 +101,6 @@ class RegressionSuite extends CatsSuite {
     Stream(1,2,6,8).traverse(validate) should === (Either.left("6 is greater than 5"))
     checkAndResetCount(3)
 
-    type StringMap[A] = Map[String, A]
-    val intMap: StringMap[Int] = Map("one" -> 1, "two" -> 2, "six" -> 6, "eight" -> 8)
-    intMap.traverse(validate) should === (Either.left("6 is greater than 5"))
-    checkAndResetCount(3)
```

Comment:
Not sure what this regression test is about, do you know? If neither of us are sure maybe we should replace it with a sortedMap one?

Reply:
Apparently it's about https://github.com/typelevel/cats/issues/513. I had to adjust this test slightly, because the order of the Strings matter now, so it wouldn't work if we just change the type of the collection :)